app/main.js

```diff
@@ -1315,26 +1317,31 @@ function getTrayMenuTemplate () {
       submenu: [
         {
           label: i18next.t('utils.minutes', { count: 30 }),
+          accelerator: settings.get('pauseBreaksFor30MinutesShortcut'),
```

Comment:
I guess not that important but returns `[18960:0723/192752.999383:WARNING:accelerator_util.cc(65)]  doesn't contain a valid key` when no value set.

Reply:
Well spotted. How do I reproduce this error? I tried to start the app (`npm run start`) with empty string `""` values for the shortcuts but I don't see this error in the terminal.

Reply:
Try with `npm run dev`, it gives more debug info.

Reply:
Fixed, the warning only appeared in Ubuntu for me but not in Windows or Mac.